about.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <title>Gastronomy Retreat</title>
+</head>
+
+<body>
```

Comment:
- [ ] Kindly fix horizontal scroll on the about page currently there is a horizontal scroll, please check the widths, padding and margins used to fix this issue, see video below 👍🏿


https://user-images.githubusercontent.com/69638013/169292534-7b496eaa-a164-46b5-abf6-eb8cef095675.mp4